scripts/transform.sh

```diff
@@ -59,5 +59,6 @@ cd "$individual_packages_folder"
 createSymlinks "$individual_packages_folder"
 
 if [ "$skip_build" != "true" ]; then
-  PHASE=transform yarn lerna run --stream $runtarget
+  CONCURRENCY=${CONCURRENCY:-4}
```

Comment:
Should we raise the value for the cdk build or is it 4 even now?

Reply:
The value is currently ♾  for the cdk build. What would you propose we set it to?

Reply:
What is the actual translate to today? The number of alpha modules? 
Perhaps that's a good start, given that the build works there and we don't want to slow it down any further.

I'm not fussed here, if you want to leave it at 4.

Reply:
The number is 26 today, which I think it unreasonably high.

I'm going to look at the PR build timings (compared to other PR builds) and see how much 4 as the default slows it down. I also just pushed a change to make the default 8, and I'll see how that goes. I suspect going much higher than 8 isn't going to be much faster, due to resource contention, but it's worth a look.

Reply:
With concurrency of 8, we're within a minute of the original (unbounded) run time. I think that's a fair compromise.